scripts/dist.sh

```diff
@@ -1,6 +1,5 @@
 #!/bin/bash
 set -euo pipefail
-IFS=$"\\n\\t"
```

Comment:
I couldn't determine that this line had any affect on the functionality of the script and it was causing the `n` in `darwin` to be removed when I tried to call the script like `scripts/dist.sh darwin amd64`.

Reply:
The intention of this instruction is to allow file names with spaces in them, but in the current form there is a bug because the escaping is wrong. It should say: IFS=$'\n\t' (single quote escape)

But since we don't use file names with spaces in this script I think it's ok to delete the statement. :-)